docs/index.md

```diff
@@ -68,7 +68,7 @@ The TypeScript files are generated at compile-time.
 
 By default, the TypeScript generation is executed in directory `target/scala-ts/src_managed` (see `sourceManaged in scalatsOnCompile` in [configuration](./config.html)).
 
-> See [SBT plugin settings](./config.html#sbt-plugin-settings)
+> See [SBT plugin settings](./config.md#sbt-plugin-settings)
```

Comment:
These links are for the online documentation (rendered as HTML) : https://scala-ts.github.io/scala-ts/config.html